cinema/tests/test_movie_api.py

```diff
@@ -157,3 +159,92 @@ def test_image_url_is_shown_on_movie_session_detail(self):
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
+
+
+class UnauthenticatedMovieApiTestCase(APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+
+    def test_auth_list(self):
+        sample_movie()
+        response = self.client.get(MOVIE_URL)
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_auth_retrieve(self):
+        sample_movie()
+        response = self.client.get(MOVIE_URL, {id:1})
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_auth_create(self):
+        payload = {
+        "title": "Sample movie",
+        "description": "Sample description",
+        "duration": 90,
+        }
```

Comment:
Pilnuj odpowiednich wcięć. Uwaga w odniesieniu do wszystkich przypadków.
```suggestion
        payload = {
            "title": "Sample movie",
            "description": "Sample description",
            "duration": 90,
        }
```